components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/jetty/JettyServer.java

```diff
@@ -49,7 +49,8 @@
 
   private final String[] args;
 
-  private JettyMainThread thread;
+  // FIXME remove once adding docker connectors properly implemented
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)